src/engraving/libmscore/cmd.cpp

```diff
@@ -1049,6 +1053,13 @@ Fraction Score::makeGap(Segment* segment, int track, const Fraction& _sd, Tuplet
             tuplet = 0;
         } else {
             if (seg != firstSegment || !keepChord) {
+                if (cr->isChord()) {
+                    for (auto note : toChord(cr)->notes()) {
+                        _selection.remove(note);
+                    }
+                } else {
+                    _selection.remove(cr);
+                }
```

Comment:
This logic should probably be in Score::removeElement( ) but wanted to minimize impact